config-service/build.gradle.kts

```diff
@@ -55,6 +55,7 @@ tasks.integrationTest {
 }
 
 dependencies {
+  implementation(libs.hypertrace.grpcutils.context)
```

Comment:
Considering no source was changed in this module, shouldn't be any dep additions. Looks like maybe this was meant for the integration test?

Reply:
Right. Was looking to avoid `GrpcClientRequestContextUtil` in the newly added code as it is deprecated. Actually, let me replace all the occurrences as it isn't a large PR.

Reply:
I saw updates in tests and int tests, but no updates in the actual implementation. If I misread disregard, but if it's only being changed in tests can switch this dependency to be a testImplementation and integrationTestImplementation one.